docs/pages/eas-update/asset-selection.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Asset selection and exclusion
+description: Select which assets should be included in updates.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { GithubIcon } from '@expo/styleguide-icons';
+
+> **info** Available for SDK 50 and above ([`expo-updates`](/versions/latest/sdk/updates/) >= 0.23.0).
+
+In SDK 49 and earlier, all assets resolved in the Metro bundler are included in every update, and are uploaded to the update server. The new experimental asset selection feature in SDK 50 allows the developer to specify that only certain assets should be included in updates. This has the potential to greatly reduce the number of assets that need to be uploaded to and downloaded from the updates server. This feature will work with the EAS update server, or any custom server that complies with the [`expo-updates` protocol](/technical-specs/expo-updates-1.mdx).
+
+To use this feature, a new property `extra.updates.assetPatternsToBeBundled` should be included in your Expo config. It should define one or more file matching patterns (regular expressions). For example, suppose your `app.json` file has the patterns defined this way:
```

Comment:
let's add a note in here that this location will change in an upcoming release.

Reply:
I would prefer not to do that. If we do change the location, we will implement that change so that both the old and new locations will still work so that we don't break customers.

Reply:
Hmm. Maybe we should decide on it's final resting place now then? Having it in extra probably isn't the right end-location (we just put it there to not need to modify the manifest schema, but that means that it's not validated). IMO it should be a top-level field since it affects all `npx expo export` commands but I guess it could also be elsewhere.

Reply:
I guess it's fine to leave in extra for now, but IMO moving it later is a lot more work.